frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/internal-actions.ts

```diff
@@ -37,6 +37,27 @@ export const vmWizardInternalActions: VMWizardInternalActions = {
     },
     type: InternalActionType.Update,
   }),
+  [InternalActionType.UpdateUserTemplateName]: (id, value) => ({
```

Comment:
that is too many actions and it will not scale. Please create just a generic action like `setCommonDataDataValue`

Reply:
and it seems you are dispatching it in just one place, so you don't need such granularity